### ISSUE TITLE
Iterative model development and model retrofitting

### ISSUE THREAD

**Issue by KennethEnevoldsen** [centre-for-humanities-computing/danish-foundation-models#136]
In line with the [blog post](https://colinraffel.com/blog/a-call-to-build-models-like-we-build-open-source-software.html) by Collin Raffel - we have a number of possible options to improve the performance of existing model:

- [ ] E.g. adding [SWA](https://pytorch.org/blog/stochastic-weight-averaging-in-pytorch/) 
- [ ] Or implementing ELECTRA style pre-trained using the trained model (outlined in the ELECTRA paper)
- [ ] Retro-fitting a larger model vocabulary (e.g, using the Unigram)
- [ ] Converting the model to mixed precision to allow quicker download. Will probably require some post-training.

**Comment by saattrupdan** [comment]
We could also see if we could implement [this technique](https://arxiv.org/abs/1905.12801), which reduces gender bias by modifying the loss function.
